app/src/main/java/com/github/sdp/ratemyepfl/utils/RepositoryUtils.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.sdp.ratemyepfl.utils
+
+import com.google.firebase.firestore.CollectionReference
+import com.google.firebase.firestore.DocumentSnapshot
+import com.google.firebase.firestore.QuerySnapshot
+import kotlinx.coroutines.tasks.await
+
+class RepositoryUtils {
+
+    companion object {
+
+        suspend fun getLimit(collection: CollectionReference, limit: Long): QuerySnapshot {
```

Comment:
Instead of a new file, you could put these methods in the super class Repository maybe

Reply:
Makes more sense, thank you.